jgrapht-core/src/main/java/org/jgrapht/experimental/alg/color/BrownBacktrackColoring.java

```diff
@@ -28,12 +28,15 @@
  * @param <V> the graph vertex type
  * @param <E> the graph edge type
  * 
- * @author micha
+ * @author Michael Behrisch
  */
 public class BrownBacktrackColoring<V, E>
```

Comment:
This class needs some love (possibly in a follow-up pull request). Here are several considerations:
1. Algorithm needs to implement new GraphColoringAlgorithm interface
2. Should be moved out of experimental package (1)
3. I believe that the implementation of this class is based on "Brown, J. R., Chromatic scheduling and the chromatic number problem, Management Science 19 (1972), 456-463."
a. a reference to this should be added.
b. Later, in "Brèlaz, D., New methods to color the vertices of a graph, Communications of the Assoc. of Comput. Machinery 22 (1979), 251-256." apparently an improved version of this algorithm has been published. Since both papers are pretty old, there are most likely alternative algorithms available.
4. should we delete the interfaces "ExactAlgorithm" and "ApproximationAlgorithm"? Thus far we haven't used them anywhere else.

Let's add the above 4 items as "todo's" in the javadoc of this class, otherwise I'll forget about them. 


(1) I'm in favor of eventually deleting the entire experimental package. This package gives the impression that it contains 'half finished'/'poorly tested'/'possibly buggy' implementations. This is not the appearance we want.